File: 프로그래머스/1/12919. 서울에서 김서방 찾기/서울에서 김서방 찾기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(vector<string> seoul) {
    
    string kim = "Kim";
    
    int index = 0;
    vector<string>::iterator ptr;
    for(ptr=seoul.begin(); ptr != seoul.end(); ptr++){
        if ((*ptr).compare(kim) == 0) break;
        index += 1;
    }
    
    string answer = "김서방은 " + to_string(index) + "에 있다";
    return answer;
}

File: 프로그래머스/1/12919. 서울에서 김서방 찾기/서울에서 김서방 찾기.js
function solution(seoul) {
    var answer = '';
    let loaction = -1;
    
    seoul.map((item, index) => {
        if (item == "Kim") location = index;
    })
    
    answer = "김서방은 " + location.toString() + "에 있다";
    
    return answer;
}